storage/buffer.go

```diff
@@ -26,9 +26,25 @@ type BufferedSeriesIterator struct {
 	ok       bool
 }
 
+type nopIterator struct{}
+
+func (nopIterator) At() (int64, float64) { return math.MinInt64, 0 }
+func (nopIterator) Seek(t int64) bool    { return false }
+func (nopIterator) Next() bool           { return false }
+func (nopIterator) Err() error           { return nil }
+
+var nopIt = nopIterator{}
```

Comment:
You could put this in `storage/noop.go` with the other no-op implementation of the storage interfaces...

Reply:
Done.